### PR TITLE
On prem files

### PR DESCRIPTION
**Pull request by eeshansim**

### CODE REVIEW

File: connectors/on-prem-files/download-file-action.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+title: Workato connectors - On-prem files download file action
+date: 2018-12-04 06:00:00 Z
+---
+
+# On-prem files - Download file
+
+This action downloads an existing file. The output of this action is the raw contents and it supports [streaming](/features/file-streaming.md) contents of large files.
+
+![download file action](/assets/images/connectors/on-prem-files/download-file-action.png)
+*download file action*
```
Comment (graceliching) [inline connectors/on-prem-files/download-file-action.md]: *Download file action*

File: connectors/on-prem-files/list-files-action.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <tr>
+      <th width='25%'>Input field</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Folder</td>
+      <td>
+        Select the parent folder to list files from. This can be done either by selecting a folder from the pick list, or toggling the input field to text mode and providing the folder path.
```
Comment (graceliching) [inline connectors/on-prem-files/list-files-action.md]: If providing path in text mode, is the path an absolute path or just the part appended to the base path?

File: connectors/on-prem-files/download-file-action.md (hunk 1 above)
Comment (graceliching) [inline connectors/on-prem-files/download-file-action.md]: *Download file action*

File: connectors/on-prem-files/new-lines-trigger.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+title: Workato connectors - On-prem files new lines in CSV file trigger
+date: 2018-12-04 06:00:00 Z
+---
+
+# On-prem files - New lines in CSV file trigger
+
+This trigger picks up new lines added to the end of a selected CSV file. Lines from the file will be processed un batches in each job.
```
Comment (graceliching) [inline connectors/on-prem-files/new-lines-trigger.md]: in* batches

Comment (graceliching) [inline connectors/on-prem-files/new-lines-trigger.md]: in* batches

File: connectors/on-prem-files/new-lines-trigger.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    </tr>
+    <tr>
+      <td>Column delimiter</td>
+      <td>
+        Next, select a column delimiter for this selected CSV file. The delimiters available are: comma, semicolon, space and tab.
+      </td>
+    </tr>
+    <tr>
+      <td>Batch size</td>
+      <td>Determine the number of lines to process in each job. The default value is 100 and maximum is 2000.
```
Comment (graceliching) [inline connectors/on-prem-files/new-lines-trigger.md]: Is it relevant here to add how one should determine the required batch size?

File: connectors/on-prem-files/upload-file-action.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      <td>Created time</td>
+      <td>This is the time that the file was created. This value is set by your OS.</td>
+    </tr>
+    <tr>
+      <td>Last modified time</td>
+      <td>This is the last modified time of the file. This value is updated by your OS.</td>
+    </tr>
+    <tr>
+      <td>File existed before upload?</td>
+      <td>If there was a file with the <b>same name and extension</b> in the selected folder, this datapill with return <code>true</code>. Otherwise, <code>false</code>.</td>
```
Comment (graceliching) [inline connectors/on-prem-files/upload-file-action.md]: Does this true/false essentially correspond to whether you selected Yes/No to "Overwrite existing file?".

Comment (eeshansim) [reply connectors/on-prem-files/upload-file-action.md]: No, this is to reflect if an existing file was there before the upload. **Overwrite existing file?** is the flag for what to do when there is a file.

File: connectors/on-prem-files/new-lines-trigger.md (hunk 4 above)
Comment (eeshansim) [reply connectors/on-prem-files/new-lines-trigger.md]: Added

**Pull request merged**